code/code/sys/socket.cc

```diff
@@ -567,16 +568,35 @@ procTweakLoadRate::procTweakLoadRate(const int &p)
   name="procTweakLoadRate";
 }
 
+// The Target loadrate and rate of change should be set from within the mud,
+// not hard-coded
 void procTweakLoadRate::run(const TPulse &) const
 {
-  if(stats.equip <= 0.046){
-    vlogf(LOG_BUG, "procTweakLoadRate: desired load rate achieved.");
+
+  if ( stats.global_lp_target_changerate <= 0 ) return;     // If this is zero, the procedure won't change anything anyway.
+
+
+  // target_delta = (stats.global_lp_target - original_load)/(24hr/day*3600s/hr*delta_time_in_days)
+
+  // If the current loadrate is within one delta of the target, we will
+  // call it good. Clamp the load rate to target just to be safe and
+  // prevent hunting.
+  if( 
+        (stats.global_lp_target-stats.global_lp_target_changerate <= stats.equip) 
```

Comment:
The parentheses here are not needed (though you can retain them if you think it helps)

Reply:
I'll leave them for now. I've been bitten too many times by not having them and a different operator actually had higher precedence. That was a painful 4 hours of bug hunting.